src/network/core/tcp_coordinator.h

```diff
@@ -263,6 +265,28 @@ class NetworkCoordinatorSocketHandler : public NetworkTCPSocketHandler {
 	 */
 	virtual bool Receive_GC_STUN_CONNECT(Packet *p);
 
+	/**
+	 * Game Coordinator informs the client of the NewGRFs in the lookup table.
+	 * This packet is sent after a CLIENT_LISTING reqest, but before GC_LISTING.
+	 *
+	 *  uint32   Listing version.
+	 *  uint16   Number of NewGRFs in the packet, with for each of the NewGRFs:
+	 *  uint32   Lookup table index for the NewGRF.
```

Comment:
could you add some spaces before the part that is repeated? Makes it more clear what part is repeating.